tutorials/python/python-type-driven-development/tutorial.md

```diff
@@ -0,0 +1,218 @@
+---
+title: How to start using types in Python with Mypy
```

Comment:
Which one do we use for the URL? The filename or the title? If it's the filename, can you update it to match the title or the URL that we want?

Reply:
Good catch. Updated.